### ISSUE TITLE
sealer push compressing progress 

### ISSUE THREAD

**Issue by cuisongliu** [alibaba/sealer#39]
#### What happened:

sealer push 

```
[root@iZj6c5mhum14tik63i0x98Z ~]#sealer push registry-vpc.cn-hongkong.aliyuncs.com/sealyun/kubernetes-amd64:v1.19.9
Start to Push Image registry-vpc.cn-hongkong.aliyuncs.com/sealyun/kubernetes-amd64:v1.19.9
c17aeee67cc5 Compressing 0.00b/1.00b
```

0.00b/1.00b  no change

#### Environment:
- sealer version (use `sealer version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by justadogistaken** [comment]
It is compressing Actually, but I didn't make process of compreesing an real progress, all compressing will be 1.00b.

**Comment by cuisongliu** [comment]
so , fix it logger info is better?

**Comment by justadogistaken** [comment]
leave it to me, I'll fix it later

**Comment by justadogistaken** [comment]
solved in pr [#77 ](https://github.com/alibaba/sealer/pull/77)
